proto/gz/msgs/material_color.proto

```diff
@@ -19,35 +19,53 @@
 syntax = "proto3";
 package gz.msgs;
 option java_package = "com.gz.msgs";
-option java_outer_classname = "MaterialProtos";
+option java_outer_classname = "MaterialColorProtos";
 
 /// \ingroup gz.msgs
 /// \interface MaterialColor
 /// \brief Color description for an entities link-visual material.
 
 import "gz/msgs/color.proto";
+import "gz/msgs/entity.proto";
 import "gz/msgs/header.proto";
 
 message MaterialColor
 {
   /// \brief Optional header data
   Header header          = 1;
 
-  /// \brief Name of link-visual.
-  string name            = 2;
-
-  /// \brief Name of parent enitity for link-visual.
-  string parent_name     = 3;
+  /// \brief Entity which material colors are going to be modified.
+  ///
+  /// The entity might be a model with at least one link or a link.
```

Comment:
The entity should be a visual?

Reply:
Actually I think a visual always has to be attached to a parent link. But all updated now.